packages/node/src/utils/reindex.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020-2022 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { getLogger, MmrService, StoreService } from '@subql/node-core';
+import { Sequelize } from 'sequelize';
+import { UnfinalizedBlocksService } from '../indexer/unfinalizedBlocks.service';
+import { ForceCleanService } from '../subcommands/forceClean.service';
+
+const logger = getLogger('Reindex');
+
+export async function reindex(
```

Comment:
this is not a good util function, not a pure function but depends on other services.